spec/unit/puppet/provider/rabbitmq_user/rabbitmqctl_spec.rb

```diff
@@ -3,220 +3,174 @@
-    provider.expects(:rabbitmqctl).with('delete_user', 'foo')
-    provider.destroy
-  end
-  it 'is able to retrieve admin value' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-foo [administrator]
-EOT
-    expect(provider.admin).to eq(:true)
-  end
-  it 'is able to retrieve correct admin value when there are multiple results' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-one [administrator]
-foo []
-EOT
-    expect(provider.admin).to eq(:false)
-  end
-  it 'fails if admin value is invalid' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-foo fail
-EOT
-    expect { provider.admin }.to raise_error(Puppet::Error, %r{Could not match line})
-  end
-  it 'is able to set admin value' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-foo   []
-icinga  [monitoring]
-kitchen []
-kitchen2        [abc, def, ghi]
-EOT
-    provider.expects(:rabbitmqctl).with('set_user_tags', 'foo', ['administrator'])
-    provider.admin = :true
-  end
-  it 'does not interfere with existing tags on the user when setting admin value' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-foo   [bar, baz]
-icinga  [monitoring]
-kitchen []
-kitchen2        [abc, def, ghi]
-EOT
-    provider.expects(:rabbitmqctl).with('set_user_tags', 'foo', %w[bar baz administrator].sort)
-    provider.admin = :true
-  end
-  it 'is able to unset admin value' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-foo     [administrator]
-guest   [administrator]
-icinga  []
-EOT
-    provider.expects(:rabbitmqctl).with('set_user_tags', 'foo', [])
-    provider.admin = :false
-  end
-  it 'does not interfere with existing tags on the user when unsetting admin value' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-foo   [administrator, bar, baz]
-icinga  [monitoring]
-kitchen []
-kitchen2        [abc, def, ghi]
-EOT
-    provider.expects(:rabbitmqctl).with('set_user_tags', 'foo', %w[bar baz].sort)
-    provider.admin = :false
-  end
 
-  it 'clears all tags on existing user' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-one [administrator]
-foo [tag1,tag2]
-icinga  [monitoring]
-kitchen []
-kitchen2        [abc, def, ghi]
-EOT
-    provider.expects(:rabbitmqctl).with('set_user_tags', 'foo', [])
-    provider.tags = []
+  describe '#destroy' do
+    it 'removes a user' do
+      provider.expects(:rabbitmqctl).with('delete_user', 'rmq_x')
+      provider.destroy
+    end
   end
 
-  it 'sets multiple tags' do
-    provider.expects(:rabbitmqctl).with('-q', 'list_users').returns <<-EOT
-one [administrator]
-foo []
-icinga  [monitoring]
-kitchen []
-kitchen2        [abc, def, ghi]
-EOT
-    provider.expects(:rabbitmqctl).with('set_user_tags', 'foo', %w[tag1 tag2])
-    provider.tags = %w[tag1 tag2]
-  end
+  describe '#check_password' do
```

Comment:
Is this doing anything useful? Or should this (syncing password) be checked in the type spec instead, or by calling another method vs. the check_password one?